.vscode/settings.json

```diff
@@ -115,4 +115,8 @@
     "editor.action.fixAll": true, // Run the default formatter before eslint, it handles whitespace better
     "source.fixAll.eslint": true // then run ESLint auto-fix
   },
-}
+  "gitlens.advanced.blame.customArguments": [
```

Comment:
Should we add `gitlens` as a recommended extension in `.vscode/extensions.json` ?

Reply:
Idk, I've always thought that might be a bit _too_ opinionated (especially now that they have paid options).  It's not for everybody.  But there shouldn't be any harm done by this if you don't have it installed.